.travis.yml

```diff
@@ -28,5 +29,5 @@ deploy:
   distributions: "bdist_wheel sdist"
   on:
     all_branches: true
-    condition: $DJANGO_VERSION = ">=1.10, < 1.11"
+    condition: $DJANGO_VERSION = ">=1.10, < 2.0"
```

Comment:
This is wrong - it needs to match one of the above entries.

e.g. here it should be `condition: $DJANGO_VERSION = ">=1.11, < 2.0"`

Reply:
Fixed that